pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -409,6 +409,9 @@ let
 
           # Stop lib depending on the -dev output of llvm
           remove-references-to -t ${llvmPackages.llvm.dev} "$out/lib/llvmjit${dlSuffix}"
```

Comment:
This doesn't evaluate for at least static.
Maybe we should turn the entire jit stuff off in this case? Doesn't work anyways then.